File: src/main/resources/output/IFC4X3_python/Entities/IfcRoot/IfcPropertyDefinition/IfcPropertyTemplateDefinition/IfcPropertyTemplate/IfcSimplePropertyTemplate/IfcSimplePropertyTemplate.py
class IfcSimplePropertyTemplate(IfcPropertyTemplate):
    def __init__(self):
        self.TemplateType: IfcSimplePropertyTemplateTypeEnum = None
        self.PrimaryMeasureType: IfcLabel = None
        self.SecondaryMeasureType: IfcLabel = None
        self.Enumerators: IfcPropertyEnumeration = None
        self.PrimaryUnit: IfcUnit = None
        self.SecondaryUnit: IfcUnit = None
        self.Expression: IfcLabel = None
        self.AccessState: IfcStateEnum = None

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcPropertyDefinition/IfcPropertyTemplateDefinition/IfcPropertyTemplate/IfcSimplePropertyTemplate/IfcSimplePropertyTemplate.vb
Public Class IfcSimplePropertyTemplate Inherits IfcPropertyTemplate
    Public TemplateType As IfcSimplePropertyTemplateTypeEnum
    Public PrimaryMeasureType As IfcLabel
    Public SecondaryMeasureType As IfcLabel
    Public Enumerators As IfcPropertyEnumeration
    Public PrimaryUnit As IfcUnit
    Public SecondaryUnit As IfcUnit
    Public Expression As IfcLabel
    Public AccessState As IfcStateEnum
End Class
